e2e/playwright/copy-move-actions/src/tests/destination-picker-dialog.test.ts

```diff
@@ -0,0 +1,107 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ApiClientFactory, MyLibrariesPage, NodesApi, SitesApi, test, Utils } from '@alfresco/playwright-shared';
+import { expect } from '@playwright/test';
+import { Site } from '@alfresco/js-api';
+import { Logger } from '@alfresco/adf-testing';
```

Comment:
Not sure if that is only for me, can you see the same warning? If yes can you try to fix this?
<img width="1325" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/d3a4ea65-c514-449a-b7f6-a83b82b8e2d9">

Reply:
@swapnil-verma-gl this one is not addressed. I can see that this one has the same warning for almost each e2e file, not sure why.

Reply:
Thank you @swapnil-verma-gl for investigation. Would you like to investigate it deeper and fix it as part of that PR or do you prefer raise separate jira for this?

Reply:
Ideally, we should deprecate `@alfresco/adf-testing` as soon as protractor is migrated to playwright

Reply:
In that case, I suppose we could leave this warning as is for now. It's not impacting the build, and will be removed eventually when the adf-testing library is depracated

Reply:
Raised [ACS-6442](https://alfresco.atlassian.net/browse/ACS-6442) for tracking deprecation of @alfresco/adf-testing